x/market/client/cli/txDeal.go

```diff
@@ -82,3 +109,48 @@ func parseCreateDealFlags(fs *flag.FlagSet) (*createDealInputs, error) {
 
 	return deal, nil
 }
+
+func NewSellDataMsg(clientCtx client.Context, txf tx.Factory, fs *flag.FlagSet) (tx.Factory, sdk.Msg, error) {
+	sellData, err := parseSellDataFlags(fs)
+	if err != nil {
+		return txf, nil, fmt.Errorf("failed to parse receipt: %w", err)
+	}
+
+	unSigned := types.UnsignedDataValidationCertificate{
+		DealId:               sellData.Cert.UnsignedCert.DealId,
+		DataHash:             sellData.Cert.UnsignedCert.DataHash,
+		EncryptedDataUrl:     sellData.Cert.UnsignedCert.EncryptedDataUrl,
+		DataValidatorAddress: sellData.Cert.UnsignedCert.DataValidatorAddress,
+		RequesterAddress:     sellData.Cert.UnsignedCert.RequesterAddress,
+	}
+
+	signed := types.DataValidationCertificate{
+		UnsignedCert: &unSigned,
+		Signature:    []byte(sellData.Cert.Signature),
+	}
+
+	msg := types.NewMsgSellData(signed, clientCtx.GetFromAddress().String())
```

Comment:
If you accept the contents of the [previous PR](https://github.com/medibloc/panacea-core/pull/255/files#r788414624), this part should also be modified.